libraries/core_libs/node/include/node/node.hpp

```diff
@@ -118,4 +118,14 @@ class FullNode : public std::enable_shared_from_this<FullNode> {
   void rebuildDb();
 };
 
+class HardforkExecutors {
```

Comment:
when this is used?

Reply:
It is not used right now. I will remove it